bigtable/spark/build.sbt

```diff
@@ -0,0 +1,55 @@
+name := "bigtable-spark-samples"
+
+version := "0.1"
+
+scalaVersion := "2.11.12"
+
+// Versions to match Dataproc
```

Comment:
which version of data proc?

Reply:
As discussed offline, we will target dataproc 1.4

According to:
https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-release-1.4

sparkVersion = 2.4.6
hbaseVersion = 1.3.6
scala = 2.11.12